spec/fetch_spec.js

```diff
@@ -89,6 +89,7 @@ describe('fetch', () => {
 
         workspace.openIngest();
         content.actionOnItem('Fetch To', 0);
+        content.actionOnItem('Fetch To', 0);
```

Comment:
Why perform the same action multiple times?

Reply:
Fixed